content/guides/references/migration-guide.md

```diff
@@ -579,7 +578,7 @@ configuration will result in an error when Cypress loads.
 
 </Alert>
 
-### Updated test file locations
+### Updated Test File Locations
 
 Previously, you could specify the locations of test files and folders using the
 configuration options: `componentFolder`, `integrationFolder`, or `testFiles`.
```

Comment:
This one is the weird one that we used `componentFolder` + `testFiles` to load ct specs or `integrationFolder` + `testFiles` to load e2e tests. Probably the reason 'redundant' was referenced above 

```suggestion
configuration options: `componentFolder`, or `integrationFolder`, and `testFiles`.
```